packages/dapp/package.json

```diff
@@ -7,44 +7,44 @@
     "@arcxmoney/analytics": "^1.9.1",
     "@emotion/react": "^11.11.1",
     "@emotion/styled": "^11.11.0",
-    "@lifi/sdk": "^2.2.0",
-    "@lifi/wallet-management": "^2.2.1",
-    "@lifi/widget": "^2.2.1",
-    "@mui/icons-material": "^5.14.0",
-    "@mui/material": "^5.14.0",
+    "@lifi/sdk": "^2.2.2",
+    "@lifi/wallet-management": "^2.2.4",
+    "@lifi/widget": "^2.2.6",
+    "@mui/icons-material": "^5.14.3",
+    "@mui/material": "^5.14.3",
     "@safe-global/safe-apps-sdk": "^8.0.0",
-    "@sentry/browser": "^7.58.1",
-    "@sentry/integrations": "^7.58.1",
-    "@sentry/react": "^7.58.1",
-    "@tanstack/react-query": "^4.29.19",
+    "@sentry/browser": "^7.61.0",
+    "@sentry/integrations": "^7.61.0",
+    "@sentry/react": "^7.61.0",
+    "@tanstack/react-query": "^4.32.1",
     "@transferto/shared": "*",
-    "@types/react": "^18.2.15",
-    "@widgetbot/react-embed": "^1.7.0",
+    "@types/react": "^18.2.18",
+    "@widgetbot/react-embed": "^1.8.0",
     "ethers": "^5.7.2",
-    "i18next": "^22.5.0",
-    "i18next-browser-languagedetector": "^7.0.2",
+    "i18next": "^22.5.1",
+    "i18next-browser-languagedetector": "^7.1.0",
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
     "react-ga4": "^2.1.0",
-    "react-hotjar": "^5.5.0",
+    "react-hotjar": "^6.1.0",
     "react-i18next": "^12.3.1",
     "react-query": "^3.39.3",
-    "react-router-dom": "^6.14.1",
+    "react-router-dom": "^6.14.2",
     "typescript": "^5.1.6",
     "use-media": "^1.4.0",
```

Comment:
`use-media` can be deleted from here as well

Reply:
Nice, use-media is removed as well now